### DISCUSSION TITLE
Compatibility Ceph 18.2.0 with OKD

### DISCUSSION THREAD

**Discussion by dborgi** [okd-project/okd#1713] [General]
Hello

    I installed ceph 18.2.0 to make Persistent Volume with Redhat Logging operator and Elasticsearch

![image](https://github.com/okd-project/okd/assets/48066051/4c3e731f-ab28-48b6-863d-a72ef6f5602d)

My OKD is version 4.13 and I installed with helm the rbd.csi.ceph.com driver and configured the StorageClass but it does not bound

 My question is if I'm configuring it wrong or it doesn't support the version of ceph for OKD or what driver i need to install to work it

**Comment by ibotty** [comment]
I am running 18.2.0 managed by rook on multiple clusters without any problems.

**Reply by dborgi** [reply]
Do you have any guide to configure it?

**Comment by peterroth** [comment]
It might be better to ask this question at Rook's side, the issue probably comes from your configuration, for example how you defined where the ODSs should run, did you dedicate any block device (sdb, sdc, etc) for the OSDs to use, such things. If you run `oc get pods -n rook-ceph`, do you see every required pod is up and running? If you deploy the  [rook-ceph-toolbox](https://rook.io/docs/rook/v1.12/Troubleshooting/ceph-toolbox/#interactive-toolbox) and get into it, run `ceph status`, what does it say?
